pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1022,14 +1052,14 @@ func (l *LibvirtDomainManager) SyncVMI(vmi *v1.VirtualMachineInstance, useEmulat
 			return nil, err
 		}
 		logger.Info("Domain started.")
-	} else if cli.IsPaused(domState) {
+	} else if cli.IsPaused(domState) && !l.paused.contains(vmi.UID) {
```

Comment:
what is the `l.paused` map book keeping protecting here? I'm not entirely sure I understand why the original `if cli.IsPaused()` statement existed to begin with.

Reply:
I understood that qemu can pause VMs under some circumstances without the user requesting it. SyncVMI then should try to unpause.
In order to prevent unpausing VMs which were paused by users, I keep track of them in the paused struct.
While writing this I'm wondering if it would be enough to check for the `ReasonPausedUser` state change reason on the domain...?